vendor/bat-native-ads/src/bat/ads/internal/ml/pipeline/pipeline_embedding_info.cc

```diff
@@ -0,0 +1,38 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/ml/pipeline/pipeline_embedding_info.h"
+
+#include <utility>
+
+namespace ads {
+namespace ml {
+namespace pipeline {
+
+PipelineEmbeddingInfo::PipelineEmbeddingInfo() = default;
+
+PipelineEmbeddingInfo::PipelineEmbeddingInfo(
+    PipelineEmbeddingInfo&& info) noexcept = default;
+
+PipelineEmbeddingInfo& PipelineEmbeddingInfo::operator=(
+    PipelineEmbeddingInfo&& info) noexcept = default;
+
+PipelineEmbeddingInfo::~PipelineEmbeddingInfo() = default;
+
+PipelineEmbeddingInfo::PipelineEmbeddingInfo(
+    const int version,
+    const std::string& timestamp,
```

Comment:
could not use base::Time and then convert to a string if/when needed?

Reply:
Would be good to move away from the string. As recently we removed timestamps as string/int64_t in favor of base::Time

Reply:
Updated